evaluation.md

```diff
@@ -14,6 +14,18 @@ describe_ why _you made the change._
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
+The first issue I ran into was that I decided I needed a way to say what state it currently is in. "Staying" implies you have the same state, but when "turning" you need to say from which state to the other. If you don't move, but need to change state, I used the language "get off" and "get on" to indicate it's switching routes.
+
+Similarly, I needed a way to make sure I knew which direction was blocked. There are a few ways I could probably have approached this, but for now am just adding in a direction.
+
+The last thing I needed to hammer out was how to handle surroundings information when there was two-four different things someone is specifying. I debated whether or not to make that explicit, or if you could leave it out or just say "hey the user isn't allowed to do that in this language" but I decided I'd rather make the language a bit more awkward and instead keep that functionality. To handle this, I just expanded the "when the X road ends" statement, so that if it's blocked the X road has ended, you can implicitly say that a road is open by saying to turn (right now assuming that that error would get cauught later, and not in the design of the language) and then if a secondary/tertiary/etc road is blocked, it "is closed." I also at this point decided explicitly that you can only have one open direction, it's just whether or not you also specificy one or more blocked roads. Since picobot can only move in one direction, it seems overkill to account for whether more than one road is unblocked explicitly. Similarly, I'm not accounting for where there are all four roads "closed"
```

Comment:
I think you made the right call with the "X road ends and Y road is closed" syntax. Some might prefer and'ing them all together like "X road and Y road ends" but that would violate the driving down one path paradigm you're working with here, so for that reason I prefer what you did